lib/openai.ex

```diff
@@ -198,6 +199,50 @@ defmodule OpenAI do
     Classifications.fetch(params)
   end
 
+
+  @doc """
+  Gets info about the fine-tune job.
+  ## Example request
+      OpenAI.fetch("ft-")
```

Comment:
@mrmrinal thanks a lot for your contribution! I think that this line of doc should be `OpenAI.finetunes("ft-")` instead of `OpenAI.fetch("ft-")`. Can you please correct it? Then we can merge and release the new version including your code 👍

Reply:
@mgallo I have made the changes to the documentation :)